### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by JohanPetersson**

### CODE REVIEW

File: src/@episerver/test-setup/backend/Backend.csproj [hunk 1]
```diff
@@ -6,10 +6,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="EPiServer.CMS" Version="12.0.2" />
-    <PackageReference Include="EPiServer.ContentDefinitionsApi" Version="3.0.0-ci-002490" />
-    <PackageReference Include="EPiServer.ContentDeliveryApi.Cms" Version="3.0.0-ci-002490" />
-    <PackageReference Include="EPiServer.OpenIDConnect" Version="3.0.0-ci-002490" />
+    <PackageReference Include="EPiServer.CMS" Version="12.1.0" />
+    <PackageReference Include="EPiServer.Hosting" Version="12.0.4" />
```
Comment (pergunsarfs) [inline src/@episerver/test-setup/backend/Backend.csproj:10]: Shouldn't EPiServer.Hosting be 12.1.0 as well?

Comment (JohanPetersson) [reply src/@episerver/test-setup/backend/Backend.csproj]: Fixed

**Review by pergunsarfs — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -25,15 +27,16 @@ This project uses:
 * .NET SDK 5+
 * SQL Server 2016 Express LocalDB ([download here](https://www.microsoft.com/en-us/sql-server/sql-server-downloads))
 
-## Creating a release
+## Create a release
```
Comment (tidyui) [inline README.md:30]: Should this information be in the public README at all, maybe it's just confusing and gives contributors the impression that they need to do this when submitting a PR to the repository?

Comment (JohanPetersson) [reply README.md:30]: I rather have it here than elsewhere. Can add a note that only administrators have permission to create the actual release on step 4?

**Pull request merged**